### PR TITLE
Pie

### LINKED ISSUE #4559: [Term Entry] Python Plotly .pie()
### Reason/inspiration (optional)

We would like a new term entry in the `Data Visualization` concept for Plotly: `.pie()`. The entry should go in a new file under `docs/content/plotly/concepts/express/terms/pie/pie.md`

The entry should describe the term and how it is used. Also include:

- An introductory paragraph that describes the concept
- A 'Syntax' section that includes the syntax for the concept (if applicable)
- An 'Example' section that shows the concept in use, include an output image as well.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by jmarsden8**
### Description

Created plotly pie sheet.

### Issue Solved
Closes #4559 

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/plotly/concepts/express/terms/pie/pie.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Plotly Pie Charts
+
+## .pie()
```
Comment (mamtawardhani) [inline content/plotly/concepts/express/terms/pie/pie.md:3]:
```suggestion
---
Title: '.pie()'
Description: 'Creates a pie chart, a circular statistical graphic divided into slices to illustrate numerical proportions.'
Subjects:
  - 'Data Science'
  - 'Data Visualization'
Tags:
  - 'Graphs'
  - 'Libraries'
  - 'Plotly'
  - 'Data'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---
```

Comment (mamtawardhani) [reply content/plotly/concepts/express/terms/pie/pie.md:3]: Meta data is needed at the start of every entry

File: content/plotly/concepts/express/terms/pie/pie.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+In python, we can use *.pie()* from the *plotly library* to display data in the form of a pie chart.
+This is such a simple tool to use and can transform data into visually apealing ways for readers
+to get a better idea about the relationships between variables.
```
Comment (mamtawardhani) [inline content/plotly/concepts/express/terms/pie/pie.md:7]:
```suggestion
In Python, the **`pie()`** method from the Plotly library is used to create a pie chart, which visually represents data proportions. This method helps transform data into an appealing visual format, making it easier for readers to understand the relationships between variables.
```

Comment (mamtawardhani) [reply content/plotly/concepts/express/terms/pie/pie.md:7]: Reframed the sentence for better clarity. 
- Avoid using "we", "you" in the sentences.
- The first instance of the term should be in bold and backticks: **`.pie()`**

File: content/plotly/concepts/express/terms/pie/pie.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
```
Comment (mamtawardhani) [inline content/plotly/concepts/express/terms/pie/pie.md:8]: After the introduction paragraph, use the `## Syntax` block to provide the syntax of the `.pie()` method. Additionally, describe the parameters in the syntax in the following format:
- `parameter1`: Its description
- `parameter2`: its description
- `parametern`: its description

File: content/plotly/concepts/express/terms/pie/pie.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+3. Next, we need to use .pie() on px and pass our data as the arguement for the values and names as the value for names. We create a variable called fig to hold this for later:
+```
+fig = px.pie(values=data, names=names)
+```
+
+4. Finally, we need to show our data visualization, we can do this by using .show() on the fig variable we created earlier:
+```
+fig.show()
+```
```
Comment (mamtawardhani) [inline content/plotly/concepts/express/terms/pie/pie.md:45]: These explanations can be added in the comments of the code itself

File: content/plotly/concepts/express/terms/pie/pie.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# Plotly Pie Charts
+
+## .pie()
+
+In python, we can use *.pie()* from the *plotly library* to display data in the form of a pie chart.
+This is such a simple tool to use and can transform data into visually apealing ways for readers
+to get a better idea about the relationships between variables.
+
+## Example
+```
```
Comment (mamtawardhani) [inline content/plotly/concepts/express/terms/pie/pie.md:10]:
```suggestion
## Example

The example below shows the usage of the `.pie()` method as follows:

```py
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
@jmarsden8 Thank you for your contribution to the Codecademy Docs, the entry is nicely written! 😃 

I've suggested a few changes in the entry, could you please review and modify those at your earliest convenience? Also, here is a sample entry to highlight the `Syntax` and the parameter description part that I've mentioned in the comment: https://github.com/Codecademy/docs/pull/4789/files

Thank you! 😄

**Comment by jmarsden8** [comment]
> Hey @jmarsden8! Could you please provide some insight into why the PR was closed?
> Thank you!

Very sorry, that was unintentional. I will ensure this is sorted as soon as possible.
Thanks

**Pull request closed without merging**